app/views/admin/articles/index.html.erb

```diff
@@ -21,17 +30,28 @@
       </h1>
     <% end %>
   </div>
+
+  <div class="col-12">
+    <% if @pinned_article.present? %>
+      <h2><%= image_tag("pin.svg") %> Pinned Article</h2>
```

Comment:
we should add `alt=""` to this image tag, since it's decorative and not currently populated with any alt text

(I _think_ this is the correct syntax 🙃)

```suggestion
      <h2><%= image_tag("pin.svg", alt: "") %> Pinned Article</h2>
```